packages/vanilla/src/components/global-nav/side-nav/group/module/module.js

```diff
@@ -15,9 +15,6 @@ import './module.scss';
 class Module extends Core {
   constructor(options) {
     super(options);
-    if (!options.icon) {
```

Comment:
Fixes https://github.com/Autodesk/hig/issues/596, which seems to be a regression from moving away from `options` as the way to map props.